### PR TITLE
fix: avoid `Argument list too long` error

### PR DESCRIPTION
**Pull request by richardlau**
Pass POST data to curl via a file to avoid getting an
`Argument list too long` error.

Fixes: https://github.com/nodejs/reliability/issues/211

### CODE REVIEW

**Comment by richardlau** [comment]
Was hoping to be able to manually run to test if this works with the workflow dispatch set 😞.

**Review by mhdawson — APPROVED** [review]
LGTM

**Comment by mhdawson** [comment]
I'm suggesting a fast-track on this one.

File: .github/workflows/reliability_report.yml [hunk 1]
```diff
@@ -9,27 +9,27 @@ name: Update CI reliability
...
   schedule:
     - cron: '0 0 * * *'
+  workflow_dispatch:
   
 jobs:
   create-report:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/setup-node@v1
+    - uses: actions/setup-node@v3.0.0
```
Comment (richardlau) [inline .github/workflows/reliability_report.yml:18]:
```suggestion
    - uses: actions/setup-node@v3
```
https://github.com/actions/setup-node/issues/425

**Pull request merged**